src/containers/org/OrgDataSetContainer.js

```diff
@@ -93,11 +95,17 @@ const OrgDataSetContainer = ({
   const location = useLocation();
   const organization :?Organization = useSelector(selectOrganization(organizationId));
   const dataSet :Map = useSelector(selectOrgDataSet(organizationId, dataSetId));
+  const dataSetEntityTypeId :UUID = dataSet.get(ENTITY_TYPE_ID);
   const dataSetColumns :Map<UUID, Map> = useSelector(selectOrgDataSetColumns(organizationId, dataSetId));
   const dataSetSchema :?string = useSelector(selectDataSetSchema(dataSetId));
   const dataSetSize :?number = useSelector(selectOrgDataSetSize(organizationId, dataSetId));
+  const dataSetEntityType :?EntityType = useSelector(selectEntityType(dataSetEntityTypeId));
   const getCollabWithDataSetRS :?RequestState = useRequestState([COLLABORATIONS, GET_COLLABORATIONS_WITH_DATA_SETS]);
 
+  const entityTypeString :?string = dataSetEntityType?.type
+    ? `${dataSetEntityType?.type.namespace}.${dataSetEntityType?.type.name}`
+    : null;
```

Comment:
`type` is an FQN so you don't need to manually construct the string ... just call `.toString()`. However, I would simplify this by removing this assignment and making the change below.

```suggestion
```